_posts/2018-02-26-success-with-serverless.md

```diff
@@ -13,12 +13,12 @@ tags:
 
 ### The Problem
-[AGRC's base maps]({{ "/data/base-map-and-imagery/" | prepend: site.baseurl }}) (including the [Google imagery]({{ "/discover/license/" | prepend: site.baseurl }})) are served via a custom server application called Giza. Part of the advantage of this solution is the ability to secure and track usage via quad-words. These unique words are assigned to a specific user and are locked down to a specific domain or IP address. For example, if my quad-word is locked down to <code>mydomain.com</code>, then requests originating from any other domain or IP address are blocked by the server. This prevents unauthorized access of licensed content as well as allows AGRC to track analytics.
+[AGRC's base maps]({{ "/data/base-map-and-imagery/" | prepend: site.baseurl }}) (including the [Google imagery]({{ "/discover/license/" | prepend: site.baseurl }})) are served via a custom server application called Giza. Part of the advantage of using Giza is that it allows you to secure and track usage via quad-words. These unique words are assigned to a specific user and are locked down to a specific domain or IP address. For example, if my quad-word is locked down to <code>mydomain.com</code>, then requests originating from any other domain or IP address are blocked by the server. This prevents unauthorized access of licensed content, as well as allows AGRC to track analytics.
 
-This quad-word system works great until you try to use one of Esri's out-of-the-box print services ([here's an example of one](http://mapserv.utah.gov/arcgis/rest/services/Utilities/PrintingTools/GPServer)). When you send a web map to this service, it reconstructs all of the layers on the server. This causes requests for base maps to be sent from your ArcGIS Server box, rather than your user's browser (with your domain as the referrer). We do allow wide open quad-words (not locked down to any domain/IP) to be used by those who need to make requests from servers or other local machines. However, these wide open quad-words cannot be used in web applications due to the fact that they could be copied and used by unauthorized users.
+This quad-word system works great . . . until you try to use one of Esri's out-of-the-box print services ([here's an example of one](http://mapserv.utah.gov/arcgis/rest/services/Utilities/PrintingTools/GPServer)). When you send a web map to one of these print services, the service reconstructs all of the layers on the server. This causes requests for base maps to be sent from your ArcGIS Server box rather than your user's browser (with your domain as the referrer). Now, we do allow wide-open quad-words (i.e., quad-words not locked down to any domain/IP) to be used by those who need to make requests from servers or other local machines. However, the wide-open quad-words can't be used in web applications because they could be copied and used by unauthorized users. This is a problem.
```

Comment:
_Now, we do allow wide-open quad-words (i.e., quad-words not locked down to any domain/IP) to be used by those who need to make requests from servers or other local machines_.: Did I understand the "wide-open quad-words" thing correctly? Is this rewording accurate?

Reply:
yes that is accurate.